website/server/libs/payments.js

```diff
@@ -289,7 +435,7 @@ api.cancelSubscription = async function cancelSubscription (data) {
     await data.user.save();
   }
 
-  txnEmail(data.user, emailType);
+  if (customerId !== this.constants.GROUP_PLAN_CUSTOMER_ID) txnEmail(data.user, emailType, emailMergeData);
```

Comment:
the email is only sent if this is a user subscription? because emailMergeData is not empty only `if data.groupId`